src/bun.js/api/server.zig

```diff
@@ -5077,7 +5077,7 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
             this.cached_protocol.deref();
 
             if (this.config.hostname) |host| {
-                bun.default_allocator.destroy(host);
+                bun.default_allocator.free(host);
```

Comment:
@CGQAQ 
> I cannot found any [*:0]const u8 is been passed into free as of now

Maybe it's here. `hostname` is `[*:0]const u8`.

Reply:
Oh! You're right, thanks